pkg/operator/staticpod/controller/staticpodstate/staticpodstate_controller.go

```diff
@@ -80,13 +80,15 @@ func (c *StaticPodStateController) sync(ctx context.Context, syncCtx factory.Syn
 	errs := []error{}
 	failingErrorCount := 0
 	images := sets.NewString()
+	podsFound := false
 	for _, node := range originalOperatorStatus.NodeStatuses {
 		pod, err := c.podsGetter.Pods(c.targetNamespace).Get(ctx, mirrorPodNameForNode(c.staticPodName, node.NodeName), metav1.GetOptions{})
 		if err != nil {
 			errs = append(errs, err)
 			failingErrorCount++
```

Comment:
we should distinguish between NotFound and other errors. 404 is not a reason to degrade, see logic below

Reply:
done